package.json

```diff
@@ -1,22 +1,54 @@
 {
   "name": "node-easypost",
   "description": "EasyPost Node Client Library",
-  "version": "2.1.1",
+  "version": "3.0.0",
   "author": "Sawyer Bateman <support@easypost.com>",
   "homepage": "https://easypost.com",
   "repository": {
     "type": "git",
     "url": "git://github.com/easypost/easypost-node.git"
   },
-  "main": "lib/main.js",
+  "main": "index.js",
   "engines": {
     "node": ">= v0.4.0"
```

Comment:
With the babel shim, should work anywhere; nobody should be on node 0.4 anymore, though. I'd be happy with bumping to at least 0.10. I did play around with producing a node 6+ version, which runs less compiler rules and produces a smaller and faster file, but the effects were pretty minimal. I may revisit and publish two files later on; one could include `require('node-easypost')` or `require('node-easypost/node-6`) depending.